observability/micrometer/src/test/java/org/wildfly/extension/micrometer/SubsystemParsingTestCase.java

```diff
@@ -24,7 +24,8 @@ public static Iterable<MicrometerSubsystemSchema> parameters() {
     }
 
     public SubsystemParsingTestCase(MicrometerSubsystemSchema schema) {
-        super(MicrometerExtension.SUBSYSTEM_NAME, new MicrometerExtension(), schema, MicrometerSubsystemSchema.CURRENT);
+        super(MicrometerConfigurationConstants.NAME,
+                new MicrometerExtension(), schema, MicrometerSubsystemSchema.CURRENT);
```

Comment:
Wow, aggressive line wrapping!  Are you using an 80x24 terminal? :)

Reply:
Ha! No, it is overly aggressive. Just forgot to fix that.